android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/LogTest.java

```diff
@@ -28,6 +31,9 @@
 public class LogTest extends BaseTest {
     private Context context;
 
+    @Rule
+    final private TemporaryFolder tempFolder = new TemporaryFolder();
```

Comment:
Seems like variable needs to be public. @bmeike thoughts?
```
com.couchbase.lite.LogTest > initializationError[test(AVD) - 4.4.2] [31mFAILED [0m
	org.junit.internal.runners.rules.ValidationError: The @Rule 'tempFolder' must be public.

	at org.junit.internal.runners.rules.RuleMemberValidator$MemberMustBePublic.validate(RuleMemberValidator.java:222)
```

Reply:
updated the variable with public! let me know if we need to fix this in any other way.